animations/concepts.md

```diff
@@ -0,0 +1,298 @@
+---
+layout: page
+title: "Flutter animation concepts"
+permalink: /animations/concepts.html
+---
+
+* TOC Placeholder
+{:toc}
+
+
+This section describes the basic concepts behind the Flutter animation architecture.
+
+If you use Flutter's built-in implicit and transition widgets, you may not need to interact with all of the Flutter core animation concepts directly because most of the work required to code the animation is already done. But for more complex explicit animations, it's important to understand the animation concepts used to fine-tune and control your animations.
```

Comment:
1. I'd reword this to something like:

For implicit or transition animations, you won't need to understand all of these concepts, but if you build explicit animations, you need to understand how Flutter's animation mechanism works.